### ISSUE TITLE
Darken `<dt>` labels in single degree Program at a Glance template part

### ISSUE THREAD

**Issue by cjg89** [UCF/Main-Site-Theme#360]
**Description**

See title.  Color of this text needs to be darkened slightly.

**Why it's Important**

The current color (`.text-default`/`#767676`) doesn't meet a 4.5:1 contrast ratio against the light gray background.

**Alternatives**

n/a

**Possible Implementation**

Use a color contrast checker to determine an adequately dark color, and replace usage of `.text-default` on these elements.  At least `#717171` is necessary to reach a 4.5:1 contrast ratio.

**Additional context**

We get flagged for this in SiteImprove on all degree profiles.
